src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -18,191 +19,298 @@ Usage:
     execute with root permission or use sudo before each command
 
     noobaa-cli <type> <action> [flags]
+
 `;
 
 const ARGUMENTS = `
 Arguments:
 
     <type>    Set the resource type: account, bucket, whitelist, diagnose, logging or upgrade
```

Comment:
`notification` is missing here?
@alphaprinz do you plan to update it in a separate PR?

Reply:
moving it to https://github.com/noobaa/noobaa-core/issues/8595